### PR TITLE
Convert xml labels to txt

### LINKED ISSUE #1: Convert xml files to txt
Needed to convert the provided data labels into the format YOLO expects.

XML sample file:
```
<annotation>
        <folder>FOLDER</folder>
        <filename>004233379.jpg</filename>
        <path>test/004233379.jpg</path>
        <source>
                <database>Unknown</database>
        </source>
        <size>
                <width>1080</width>
                <height>1080</height>
                <depth>3</depth>
        </size>
 <object>
                <name>9c</name>
                <pose>Unspecified</pose>
                <truncated>0</truncated>
                <difficult>0</difficult>
                <bndbox>
                        <xmin>82</xmin>
                        <ymin>723</ymin>
                        <xmax>146</xmax>
                        <ymax>752</ymax>
                </bndbox>
        </object>
 <object>
                <name>Qc</name>
                <pose>Unspecified</pose>
                <truncated>0</truncated>
                <difficult>0</difficult>
                <bndbox>
                        <xmin>86</xmin>
                        <ymin>584</ymin>
                        <xmax>155</xmax>
                        <ymax>639</ymax>
                </bndbox>
        </object>
 <object>
                <name>10h</name>
                <pose>Unspecified</pose>
                <truncated>0</truncated>
                <difficult>0</difficult>
                <bndbox>
                        <xmin>157</xmin>
                        <ymin>467</ymin>
                        <xmax>214</xmax>
                        <ymax>524</ymax>
                </bndbox>
        </object>
 <object>
                <name>10s</name>
                <pose>Unspecified</pose>
                <truncated>0</truncated>
                <difficult>0</difficult>
                <bndbox>
                        <xmin>269</xmin>
                        <ymin>378</ymin>
                        <xmax>315</xmax>
                        <ymax>443</ymax>
                </bndbox>
        </object>
 <object>
                <name>Jc</name>
                <pose>Unspecified</pose>
                <truncated>0</truncated>
                <difficult>0</difficult>
                <bndbox>
                        <xmin>395</xmin>
                        <ymin>339</ymin>
                        <xmax>433</xmax>
                        <ymax>409</ymax>
                </bndbox>
        </object>
 <object>
                <name>8h</name>
                <pose>Unspecified</pose>
                <truncated>0</truncated>
                <difficult>0</difficult>
                <bndbox>
                        <xmin>479</xmin>
                        <ymin>351</ymin>
                        <xmax>519</xmax>
                        <ymax>412</ymax>
                </bndbox>
        </object>
 <object>
                <name>4s</name>
                <pose>Unspecified</pose>
                <truncated>0</truncated>
                <difficult>0</difficult>
                <bndbox>
                        <xmin>594</xmin>
                        <ymin>405</ymin>
                        <xmax>648</xmax>
                        <ymax>457</ymax>
                </bndbox>
        </object>
 <object>
                <name>10s</name>
                <pose>Unspecified</pose>
                <truncated>0</truncated>
                <difficult>0</difficult>
                <bndbox>
                        <xmin>660</xmin>
                        <ymin>481</ymin>
                        <xmax>723</xmax>
                        <ymax>531</ymax>
                </bndbox>
        </object>
 <object>
                <name>10s</name>
                <pose>Unspecified</pose>
                <truncated>0</truncated>
                <difficult>0</difficult>
                <bndbox>
                        <xmin>505</xmin>
                        <ymin>758</ymin>
                        <xmax>568</xmax>
                        <ymax>808</ymax>
                </bndbox>
        </object>
</annotation>        

```

Sample txt file:
```
31 0.105556 0.682870 0.059259 0.026852
43 0.111574 0.566204 0.063889 0.050926
32 0.171759 0.458796 0.052778 0.052778
33 0.270370 0.380093 0.042593 0.060185
39 0.383333 0.346296 0.035185 0.064815
24 0.462037 0.353241 0.037037 0.056481
9 0.575000 0.399074 0.050000 0.048148
33 0.640278 0.468519 0.058333 0.046296
33 0.496759 0.725000 0.058333 0.046296
```

### PR DESCRIPTION
**Pull request by patrickdemers6**
Closes #1

### CODE REVIEW

**Comment by Nick-Vazquez** [comment]
You sweet sweet person.

**Comment by Nick-Vazquez** [comment]
PEP8 strikes again! Commiting cosmetic changes. Otherwise, everything looks good!

**Comment by patrickdemers6** [comment]
> PEP8 strikes again! Commiting cosmetic changes. Otherwise, everything looks good!

Does Python support adding a config file that holds our preferred formatting standards? Such as Eslint in JS.

**Comment by Nick-Vazquez** [comment]
Ya know, I'm not sure! I've always seen you do the formatting in VS Code, and I'd like to be able to do that in Python. Especially with enforcing Documentation standards

**Comment by patrickdemers6** [comment]
I don't think it gives me any credit since it's not my repo and I didn't fork.

**Comment by patrickdemers6** [comment]
For linting, we could attempt to implement https://github.com/github/super-linter

**Comment by Nick-Vazquez** [comment]
Once you get merged it's actually better because you're contributing to something that's not your own!

**Comment by patrickdemers6** [comment]
So far I'm the only contributor so technically, it's all my own :)

**Comment by Nick-Vazquez** [comment]
> For linting, we could attempt to implement https://github.com/github/super-linter

Holy Hell! I love it!!!

**Comment by Nick-Vazquez** [comment]
Ok you've worked with YOLO before, so how about I try to get the action going with all of the standards, and you work on the fancy stuff? 😂

**Comment by Nick-Vazquez** [comment]
Actually, let's move the convo to an issue or something

File: convert.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    x = x * width_percent_per_pixel
+    width = width * width_percent_per_pixel
+
+    y = y * height_percent_per_pixel
+    height = height * height_percent_per_pixel
+
+    return x, y, width, height
+
+
+def convert(filepath: str) -> None:
```
Comment (Nick-Vazquez) [inline convert.py:25]: Documentation eventually, I'd like to implement linting before that, though.

File: convert.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import glob
+from xml.dom import minidom
+from labels import get_labels
+
+
+def convert_coordinates(size, box):
```
Comment (Nick-Vazquez) [inline convert.py:6]: I'd like to have typing here

**Review by Nick-Vazquez — CHANGES_REQUESTED** [review]
Looks good! I'd like to have typing implemented in the classes, though. Makes it a lot cleaner

**Review by Nick-Vazquez — APPROVED** [review]
I like it much more now

**Pull request closed without merging**